backend/twitterbot/top_contributors.py

```diff
@@ -0,0 +1,40 @@
+import collections
+from datetime import datetime, timedelta
+
+from tournesol.models.comparisons import Comparison
+from tournesol.models.poll import DEFAULT_POLL_NAME
+from tournesol.models.ratings import ContributorRating
+
+
+def get_previous_month_top_public_contributor():
```

Comment:
If this method is able to return several users, we should use the plural.

```python
def get_last_month_top_public_contributors(poll_name: str):
```

Reply:
👍